tests/backup/backup_helper.go

```diff
@@ -2072,3 +2075,44 @@ func FetchNamespacesFromBackup(ctx context.Context, backupName string, orgID str
 	backedUpNamespaces = resp.GetBackup().GetNamespaces()
 	return backedUpNamespaces, err
 }
+
+// AreSlicesEqual verifies if two slices are equal or not
+func AreSlicesEqual(slice1, slice2 interface{}) bool {
```

Comment:
It can be simplified this way - https://go.dev/play/p/jrB5l2T-w-_x

Reply:
done

Reply:
https://jenkins.pwx.dev.purestorage.com/job/Users/job/Sumit/job/Custom-Pipelines/job/px-backup-on-demand-system-test-byoc/1074/consoleFull